h2o-py/tests/testdir_jira/pyunit_pubdev_5174_isin_efficiency.py

```diff
@@ -0,0 +1,37 @@
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+
+from h2o.exceptions import H2OResponseError
+from tests import pyunit_utils
+
+
+def pubdev_5174():
+
+    x = h2o.import_file(pyunit_utils.locate('smalldata/jira/PUBDEV-5174.csv'), header = 1)
+
+    tt = x['rr'].unique()
+    gg = tt[:10000, 0]
+
+    ww = x[~x['rr'].isin(gg['C1'].ascharacter().as_data_frame()['C1'].tolist())]
+    
+    print(x.nrow)
+    print(tt.nrow)
+    print(ww.nrow)
+
+    assert x.nrow == 1000000, "Original data has 1000000 rows"
+    assert tt.nrow == 499851, "Column rr has 499851 unique values"
+    assert ww.nrow == 979992, "Original data reduced has 979992 rows" # TODO: how can this be > 1000000 - 499851?
```

Comment:
I guess because above you do gg = tt[:10000, 0]. Most of the values are there twice, in rare cases more than two times (you have slightly below half a million of unique values). Therefore the number of reduced rows is only slightly above 20k (20008 values, that leaves 979992 rows left).